### PR TITLE
Updated commenting section

### PR DESCRIPTION
**Pull request by hermanndettmann**
I updated the section for how comments should be handled in our CSS files. Please have a look and feel free to provide critical feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,21 +81,17 @@ Component Blocks:
...
- * Purpose of the selector or the rule set
- * 1. Hardware acceleration hack
- * 2. position: sticky; on anything but top aligned elements is buggy in Chrome <37 and iOS 7+
- */
- 
+/* Rule specific comment */
 .box {
 	position: fixed;
-	transform: translate3d(0, 0, 0); /* [1] */
+	transform: translate3d(0, 0, 0); /* Hardware acceleration hack */
```
Comment (axten) [inline README.md]: can we say "enable Hardware acceleration" here? :)

File: README.md [hunk 2]
```diff
@@ -81,21 +81,17 @@ Component Blocks:
...
 	
+	/* Rule specific comment */
 	.csspositionsticky & {
-		position: sticky; /* [2] */
+		position: sticky; /* on anything but top aligned elements is buggy in Chrome <37 and iOS 7+ */
```
Comment (axten) [inline README.md]: I don't get the comment here :)
maybe we find a better example

File: README.md (hunk 1 above)
Comment (hermanndettmann) [reply README.md]: Sure!

File: README.md (hunk 2 above)
Comment (hermanndettmann) [reply README.md]: I tried to improve the wording here.

**Pull request merged**